Mathlib/GroupTheory/Index.lean

```diff
@@ -637,3 +657,12 @@ lemma card_fiber_eq_of_mem_range (f : F) {x y : M} (hx : x ∈ Set.range f) (hy
   rw [← f'.coe_toHomUnits y⁻¹, map_inv, Units.mul_inv_eq_iff_eq_mul, f'.coe_toHomUnits]
 
 end MonoidHom
+
+namespace AddSubgroup
+variable {G A : Type*} [Group G] [AddGroup A] [DistribMulAction G A]
+
+-- TODO: Why does making this lemma simp make `NumberTheory.Padic.PadicIntegers` time out?
```

Comment:
It looks like this needs fixing before the PR merges? Perhaps it's sensible to omit this change for now.

Reply:
I've moved it to #24025